### ISSUE TITLE
UIAlertController+JRAlertController.h file not found error on pod install

### ISSUE THREAD

**Issue by NishanthMurugan** [janrain/jump.ios#46]
Did the below configuration in pod to get Janrain SDK for our project. 

```
pod 'AppAuth'
pod 'Janrain', :git => 'https://github.com/janrain/jump.ios.git', :tag => '5.1'
```
**Issue 1 :**
After doing pod install, SDK prompting an error ("UIAlertController+JRAlertController.h") file not found in their library. And app failed to compile.

<img width="852" alt="screen shot 2018-01-08 at 8 34 50 pm" src="https://user-images.githubusercontent.com/30654850/34725399-cdd699f8-f576-11e7-89b3-48466b328c5c.png">

**Issue 2:** 
Did temporary local fix for the above issue by drag and drop those missing files from repo/master/Samples/SimpleCaptureDemo/SimpleCaptureDemo/Classes into pods sdk folder but it prompted following issue 

couldn't remove Janrain.framework/icons' after command failed: Directory not empty

<img width="1128" alt="screen shot 2018-01-08 at 8 42 22 pm" src="https://user-images.githubusercontent.com/30654850/34725458-fa57f332-f576-11e7-8cd2-962c0e50e745.png">
<img width="234" alt="screen shot 2018-01-09 at 7 56 06 pm" src="https://user-images.githubusercontent.com/30654850/34725511-2be76e82-f577-11e7-9c50-cfc9c4cdb187.png">

**Comment by nolanw** [comment]
@NishanthMurugan Just in case you got stuck at Issue 2: you can get past it by going to the Pods project in Xcode, finding the Janrain target, going to Build Phases, and looking for seemingly duplicate entries under "Copy Bundle Resources". Delete one of the duplicate `images` and one of the duplicate `images_bw`, then you're on your way.

**Comment by NishanthMurugan** [comment]
Thanks for suggesting the local fix @nolanw But for our production, we are having many apps which is going to use Janrain SDK, there we cant do any manual changes like we are doing it now for single app in development mode. Build will automate the pod install and push the apps to app store. So looking forward for the fix from SDK.

**Comment by nolanw** [comment]
@NishanthMurugan I hear ya, opened a ticket almost two months ago on this issue :)

There's probably a way to put a post-install hook in your Podfile to handle the missing bits. (We've forked the pod locally and fixed it there, otherwise I'd share.)

**Comment by pbjanrain** [comment]
I've logged a ticket to fix this.  If either of you want to submit a pull-request in the mean time that would help expedite things.  It's just a matter of finding the resources to test it.
Thanks

**Comment by NishanthMurugan** [comment]
Ok Thanks @nolanw  @pbjanrain

**Comment by mthormann-dhi** [comment]
@pbjanrain Any update on this fix?  Looking at the Janrain iOS install and really not wanting to have to drag the jump source into the project the old way.   Thx

**Comment by pbjanrain** [comment]
Hopefully by the end of next week.  In testing now.  We will not be addressing deprecation warnings we always maintain compatibility with the lowest supported version of iOS due to the broad install base of our customers.

**Comment by mthormann-dhi** [comment]
cool.  thx

**Comment by pbjanrain** [comment]
Hopefully these (except for deprecation warnings) have been addressed in the 5.1.1 release.  Please re-open if you find something that was missed.
